### ISSUE TITLE
Extending the Procotol for Acceleration information?

### ISSUE THREAD

**Issue by lordfolken** [Turbo87/openvario-protocol#2]
Hello, 

Can we extend the protocol to include acceleration in all axes and pitch/yaw?

  RRR.R  = roll angle in degree related to earth system,
  III.I  = pitch angle related to earth system,
  X.XX   = acceleration in X-Axis (multiples of G),
  Y.YY   = acceleration in Y-Axis,
  Z.ZZ   = acceleration in Z-Axis,

Something along the line of:
```
$POV,A,RRR.R,III.I,X.XX,Y.YY,Z.ZZ*05
```

**Comment by Turbo87** [comment]
that's certainly possible, but it should only be one `value` per `type` as described in the README

**Comment by lordfolken** [comment]
So 

A for Accerlation

$POV,A,X.XX,Y.YY,Z.ZZ*05

and 
 (Y for gYro)
$POV,Y,RRR.R,III.I*05

?

**Comment by Turbo87** [comment]
no, **one value** per type is the concept here. one for `X.XX`, one for `Y.YY`, ...

it might make sense to use a two-character scheme for the types though. maybe something like `Ax`, `Ay`, ...

**Comment by bomilkar** [comment]
I'd like to pick this up again. Also because XCSoar can make use of acceleration. Referring to:
https://forum.xcsoar.org/viewtopic.php?f=30&t=4273#p24423

Provide the 3D Acceleration **vector**. For example:
**$POV,A,-.1196,.0103,9.7923*30**
is the 3D acceleration vector in units of m/².
$POV,A,x.x,y.y,z.z*ss
x.x: in flight direction, positive is a forward acceleration increasing speed
y.y: is a sideways acceleration, positive to the right from the pilot's viewpoint
z.z: is a downward acceleration, positive is in the direction of gravity, when in normal flight

@Turbo87 it's a single value (a 3d vector) consisting of 3 scalars. Are you OK with this?
